ansible-playbook.yml

```diff
@@ -0,0 +1,62 @@
+- name: Install To-Do App on new web server
+  hosts: todo-app-group
+  remote_user: ec2-user
+  vars_prompt:
+  - name: api_key
+    prompt: api key?
+  - name: api_token
+    prompt: api token?
+  - name: organisation_id
+    prompt: organisation id?
+  - name: board_id
+    prompt: board id?
+
+
+  tasks:
+  - name: Install Git and Python 3
+    ansible.builtin.yum:
+      name: 
+      - git
+      - python
+      state: present
+    become: yes
+
+  - name: Install Poetry
+    ansible.builtin.shell:
+      cmd: curl -sSL https://raw.githubusercontent.com/python-poetry/poetry/master/get-poetry.py | python -
+      creates: ~/.poetry/bin/poetry
+
+  - name: Create To-Do App directory
+    ansible.builtin.file:
+      path: /opt/todoapp
+      state: directory
+      mode: '777'
```

Comment:
This should work but is quite permissive, it would be nice to try restricting this a bit more, for example setting the ec2-user as the owner and restricting write/execute permissions to other users;
```
      owner: ec2-user
      mode: '744'
```
I haven't tested this against your future commands so don't guarantee it works exactly as is though!